Makefile

```diff
@@ -229,10 +229,25 @@ bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metada
 	$(KUSTOMIZE) build config/manifests | $(OSDK) generate bundle $(BUNDLE_GEN_FLAGS)
 	$(OSDK) bundle validate ./bundle
 
+.PHONY: bundle-crd-only
+bundle-crd-only: manifests kustomize operator-sdk ## Generate bundle manifests for CRDs only and metadata, then validate generated files.
+	$(OSDK) generate kustomize manifests -q --interactive=false --input-dir config/manifests-crd  --output-dir config/manifests-crd
+	cd config/manifests-crd/ && $(KUSTOMIZE) edit add patch --name recipe.v0.0.0 --kind ClusterServiceVersion\
```

Comment:
Adding space after ClusterServiceVersion would be nice.

Reply:
done